### PR TITLE
Fixed GET Unapproved Data Sets bug

### PR DESCRIPTION
**Pull request by silent-po2**
Fixed GET all unapproved data sets failure and also cleaned up that code too.

### CODE REVIEW

**Review by ds161833 — APPROVED** [review]

**Comment by ds161833** [comment]
Don't forget to persist the changes that were on develop on your branch with datapoint transposition in the datasetService.tsx👍  
image
image

File: Client/src/Remote/Endpoints/DatasetEndpoint.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const dataUploadRoute = '/api/v1/dataUpload'
 const datasetRoute = '/api/v1/dataset'
 const flagDatasetRoute = '/api/v1/flagDataSet'
 const approvedDatasetRoute = '/api/v1/approveDataset'
-const unapprovedDatasetsRoute = '/api/v1/dataset/fetchUnapprovedDatasets'
+const unapprovedDatasetsRoute = '/api/v1/unapprovedDatasets'
```
Comment (GD-Dheer) [inline Client/src/Remote/Endpoints/DatasetEndpoint.tsx:14]: A+

**Review by GD-Dheer — APPROVED** [review]
Good cleanup. Looks much better

**Comment by silent-po2** [comment]
> Don't forget to persist the changes that were on develop on your branch with datapoint transposition in the datasetService.tsx👍

Thanks for the heads up, done!

**Pull request merged**